ts/components/ui/Markdown/handlers/internalLink.ts

```diff
@@ -104,7 +116,7 @@ export function handleInternalLink(
     dispatch(
       addInternalRouteNavigation({
         ...internalNavigation,
-        params: { ...internalNavigation.params, serviceId }
```

Comment:
Is it intentional?

Reply:
No, sorry! It is an error, I deleted mistakenly this parameter while investigating a bug.
Restored in [1975657](https://github.com/pagopa/io-app/pull/3257/commits/197565747971e4336376e7fbdad4b3ab2d16e9f1)